package.json

```diff
@@ -41,7 +41,7 @@
   "license": "MIT",
   "dependencies": {
     "bufferstreams": "^1.1.1",
-    "eslint": "^3.0.0",
+    "eslint": "3.x.x - 4.x.x",
```

Comment:
They have only released alpha build which means settings it to ^4.0.0 will not work. I can wait a few weeks until the official v4 is out. Thanks.